tests/Rules/PHPUnit/data/assert-same.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PHPUnit;
```

Comment:
Please use a different namespace, something like `namespace ExampleTestCase` is fine 😊

Reply:
The intention was to make it consistent with https://github.com/phpstan/phpstan-strict-rules/blob/master/tests/Rules/StrictCalls/data/strict-calls.php

Reply:
Yeah, but `PHPUnit` is a well known and used namespace :)

Reply:
ok :-)